fastlane/actions/get_latest_github_release.rb

```diff
@@ -0,0 +1,170 @@
+module Fastlane
```

Comment:
We could publish this action. It's based on `get_github_release`, but that one requires passing a tag name,